hugr-py/src/hugr/envelope.py

```diff
@@ -46,6 +46,12 @@
 # This is a hard-coded magic number that identifies the start of a HUGR envelope.
 MAGIC_NUMBERS = b"HUGRiHJv"
 
+# The all-unset header flags configuration.
+# Bit 7 is always set to ensure we have a printable ASCII character.
```

Comment:
If you can't reuse the same consts as un Rust, can you maybe add a comment about which file their Rust counterparts are defined in?